modin/polars/dataframe.py

```diff
@@ -1352,3 +1301,139 @@ def serialize(self, file=None) -> str | None:
             Serialized DataFrame.
         """
         return polars.from_pandas(self._query_compiler.to_pandas()).serialize(file)
+
+    @property
+    def style(self):
+        """
+        Create a Great Table for styling.
+
+        Returns:
+            GreatTable object.
+        """
+        return self._to_polars().style
+
+    def to_dict(
+        self, *, as_series: bool = True
+    ) -> dict[str, "Series"] | dict[str, list[Any]]:
+        """
+        Convert the DataFrame to a dictionary representation.
+
+        Args:
+            as_series: Whether to convert the columns to Series.
+
+        Returns:
+            Dictionary representation of the DataFrame.
+        """
+        if as_series:
+            return {name: self[name] for name in self.columns}
+        else:
+            return polars.from_pandas(self._query_compiler.to_pandas()).to_dict(
+                as_series=as_series
+            )
+
+    def to_dicts(self) -> list[dict[str, Any]]:
+        """
+        Convert the DataFrame to a list of dictionaries.
+
+        Returns:
+            List of dictionaries.
+        """
+        return self._to_polars().to_dicts()
+
+    def to_init_repr(self, n: int = 1000) -> str:
+        """
+        Get the string representation of the DataFrame for initialization.
+
+        Returns:
+            String representation of the DataFrame for initialization.
+        """
+        return self._to_polars().to_init_repr(n)
+
+    def to_struct(self, name: str = "") -> "Series":
+        """
+        Convert the DataFrame to a struct.
+
+        Args:
+            name: Name of the struct.
+
+        Returns:
+            Series representation of the DataFrame as a struct.
+        """
+        raise NotImplementedError("not yet")
+
+    def unpivot(
+        self,
+        on,
+        *,
+        index,
+        variable_name: str | None = None,
+        value_name: str | None = None,
+    ) -> "DataFrame":
+        """
+        Unpivot a DataFrame from wide to long format.
+
+        Args:
+            on: Columns to unpivot.
+            index: Columns to keep.
+            variable_name: Name of the variable column.
+            value_name: Name of the value column.
+
+        Returns:
+            Unpivoted DataFrame.
+        """
+        return self.__constructor__(
+            _query_compiler=self._query_compiler.melt(
+                on=on,
+                index=index,
+                var_name=variable_name,
+                value_name=value_name,
+            )
+        )
+
+    write_avro = write_clipboard = write_csv = write_database = write_delta = (
+        write_excel
+    ) = write_ipc = write_ipc_stream = write_json = write_ndjson = write_parquet = (
+        write_parquet_partitioned
+    ) = lambda *args, **kwargs: (_ for _ in ()).throw(NotImplementedError("not yet"))
```

Comment:
I didn't know you could use `generator.throw` like this

Reply:
Yes, since you cannot simply `raise` inside of a lambda this is the cleanest way to do it 😄.